src/StaticModel.js

```diff
@@ -11,6 +11,13 @@ export default class StaticModel {
     return new this
   }
 
+  static config(config) {
+    let self = this.instance()
+    self.config(config)
+
+    return self
+  }
+
```

Comment:
@JoaoPedroAS51 can you elaborate about it?

What is it use case?

Reply:
This method was already added. It's in docs, but I forgot to add a static version of the method. See https://github.com/robsontenorio/vue-api-query/blob/dev/src/Model.js#L40

Reply:
Got i!